doc/modules/linear_model.rst

```diff
@@ -126,6 +126,25 @@ its ``coef_`` member::
     >>> reg.intercept_
     0.13636...
 
+Note that the class :class:`Ridge` allows for the user to specify that the 
+solver be automatically chosen when ``solver=auto``. When this option 
```

Comment:
"auto" is a string and I believe that we no longer need to double quote technical word in our restructured text formatted doc.

```suggestion
solver be automatically chosen when `solver="auto"`. When this option 
```